locales/pl.json

```
@@ -2,120 +2,136 @@
     },
     "aria-meter-name": {
-      "description": "Każdy licznik (meter) oparty na ARIA ma dostępną nazwę.",
-      "help": "Liczniki (meter) ARIA muszą mieć dostępną nazwę."
+      "description": "Każdy element ARIA meter (licznik) ma dostępną nazwę.",
+      "help": "Liczniki ARIA (meter) muszą mieć dostępne nazwy."
     },
     "aria-progressbar-name": {
-      "description": "Każdy pasek postępu (progressbar) ARIA ma dostępną nazwę.",
-      "help": "Paski postępu (progressbar) ARIA muszą mieć dostępną nazwę."
+      "description": "Każdy element ARIA progressbar (pasek postępu) ma dostępną nazwę.",
+      "help": "Paski postępu ARIA (progressbar) muszą mieć dostępne nazwy."
+    },
+    "aria-prohibited-attr": {
+      "description": "Upewnij się, że atrybuty ARIA nie są zabronione dla roli elementu.",
+      "help": "Elementy mogą używać tylko dozwolonych atrybutów ARIA."
     },
     "aria-required-attr": {
-      "description": "Elementy z rolami ARIA mają wszystkie wymagane atrybuty ARIA.",
+      "description": "Elementy z rolą ARIA mają wszystkie wymagane atrybuty aria-*",
       "help": "Wymagane atrybuty ARIA muszą istnieć."
     },
     "aria-required-children": {
-      "description": "Elementy z rolą ARIA, które wymagają ról potomnych, zawierają je.",
+      "description": "Elementy z atrybutem ARIA role, które muszą zawierać elementy potomne z wymaganym atrybutem role, zawierają je.",
       "help": "Niektóre role ARIA muszą obejmować określone dzieci."
     },
     "aria-required-parent": {
-      "description": "Elementy z rolą ARIA, które wymagają ról nadrzędnych, są zawarte w elementach z takimi rolami.",
-      "help": "Niektóre role ARIA muszą być wewnątrz określonych elementów rodziców."
+      "description": "Elementy z atrybutem ARIA role, które wymagają elementu rodzica z atrybutem role, są zawarte w elementach z takimi rolami.",
+      "help": "Elementy z niektórymi atrybutami role ARIA muszą znajdować się wewnątrz nadrzędnego elementu rodzica z wymaganym atrybutem role."
     },
     "aria-roledescription": {
       "description": "Atrybut aria-roledescription jest używany tylko w elementach, które mają rolę określoną domyślnie lub jawnie.",
       "help": "Użyj aria-roledescription w elementach o roli semantycznej."
     },
     "aria-roles": {
-      "description": "Wszystkie elementy z atrybutem roli używają prawidłowej wartości tego atrybutu.",
-      "help": "Stosowane role ARIA muszą być zgodne z obowiązującymi wartościami."
+      "description": "Wartości atrybutu role są poprawne.",
+      "help": "Stosowane role ARIA muszą mieć poprawne wartości."
     },
     "aria-text": {
       "description": "Atrybut \"role=text\" jest używany dla elementów, które nie mają potomków przyjmujących fokus",
-      "help": "\"role=text\" nie powinien mieć potomków przyjmujących fokus."
+      "help": "Element z atrybutem \"role=text\" nie może mieć potomków przyjmujących fokus."
```

That should probably be like this. We only recently caught this issue in the original.

```suggestion
      "help": "Element z atrybutem role=\"text\" nie może mieć potomków przyjmujących fokus."
```